docs/how-to-develop-scraper.md

```diff
@@ -0,0 +1,225 @@
+# How To: Developer a New Scraper
+
+## 1. Find a website
+
+If you have found a website you want to scrape the recipes from, first of all check to see the website is already supported.
+
+The project [README](../README.rst) has a list of the hundreds or websites already supported.
```

Comment:
We might want to link to the fully-qualified README URL ( https://github.com/hhursev/recipe-scrapers/blob/main/README.rst ) here - because if these docs are built and deployed somewhere in future, that relative URL might not work (most of the others should.. but this one is outside the `docs` directory)

Reply:
Good catch